src/Components/OATGraphViewer/OATGraphViewer.tsx

```diff
@@ -84,24 +84,28 @@ const getGraphViewerElementsFromModels = (models, modelPositions) => {
                     new ElementEdgeData(
                         `${input['@id']}${OATComponentHandleName}${content['schema']}`,
                         content['name'],
-                        content['name'],
+                        '',
                         OATComponentHandleName
                     )
                 );
                 relationships = [...relationships, componentRelationship];
             } else if (content['@type'] === OATRelationshipHandleName) {
+                const relationshipAmount = relationships.filter(
+                    (element) =>
+                        element.source && element.source === input['@id']
+                ).length;
```

Comment:
I thought the value after the `;` was version?

Shouldn't `@id` be unique? Do we need to add an int here?